### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by SevoMkrtchyan**

### CODE REVIEW

File: src/main/java/com/example/engine/XmlConfigurationBeanScanner.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import javax.xml.XMLConstants;
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import java.io.File;
+import java.io.IOException;
+import java.util.LinkedList;
+import java.util.List;
+
+public class XmlConfigurationBeanScanner implements BeansScanner {
```
Comment (SevakMart) [inline src/main/java/com/example/engine/XmlConfigurationBeanScanner.java]: BeanDefinitionReader ?

File: src/main/java/com/example/components/BeanConfigurator.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package com.example.components;
+
+import com.example.engine.ApplicationContext;
+
+import java.util.List;
+
+public interface BeanConfigurator {
+
+    default <T> void startupBeanConfigurator(List<String> beans, ApplicationContext context) {}
```
Comment (SevakMart) [inline src/main/java/com/example/components/BeanConfigurator.java]: why this is a default if it does not have an implementation?

File: src/main/java/com/example/engine/XmlConfigurationBeanScanner.java (hunk 1 above)
Comment (SevoMkrtchyan) [reply src/main/java/com/example/engine/XmlConfigurationBeanScanner.java]: ha)

File: src/main/java/com/example/components/BeanConfigurator.java (hunk 2 above)
Comment (SevoMkrtchyan) [reply src/main/java/com/example/components/BeanConfigurator.java]: naxnakan spes ereci or ashxater minchev veranayei skilleton-@

**Pull request merged**